modules/tinymce/src/themes/silver/main/ts/ui/general/AlertBanner.ts

```diff
@@ -26,27 +28,26 @@ export const renderAlertBanner = (spec: AlertBannerWrapper, providersBackstage:
       {
         dom: {
           tag: 'div',
-          classes: [ 'tox-notification__icon' ]
+          classes: [ 'tox-notification__icon' ],
+          innerHtml: !spec.url ? icon : undefined
```

Comment:
Just wanted to sanity check that we don't the standalone icon to have a tooltip?

Reply:
I played around with `Icons.render()` to replace `Icon.get()` and noticed no difference in terms of behaviour and tests still pass except for AlertBannerTest of course. The difference is `Icons.render()` returns the icon wrapped inside an element, which is specified in one of the arguments. I don't have a strong opinion on which should be used.

Reply:
I created a follow-up ticket for investigation later so we don't block this PR being merged for the coming release